### PR TITLE
Add glue option to combine()

### PR DESCRIPTION
**Pull request by bragostin**
This is the pull request solving issue #533

### CODE REVIEW

**Comment by bragostin** [comment]
Codecov complains that "Added line #L2874 was not covered by tests" in cadquery/cq.py#L2874
even though I added in line 2168 of `tests/test_cadquery.py`
```python
        objects1.combine(glue=True, tol=None)
        self.assertEqual(11, objects1.faces().size())
```
to cover the new `glue=True` option in `combine()`.
Any idea what is missing?

**Comment by jmwright** [comment]
@bragostin Unless it's something serious, I take what codecov says with a grain of salt (not too seriously). It often complains about things that are either non-issues or not anything major.

**Comment by adam-urbanczyk** [comment]
Codecov does not make that kind of mistakes AFAIK. Probably you have only one item (cq. Compound) in objects. Just put a breakpoint in this test and verify yourself.

**Comment by bragostin** [comment]
You are right @adam-urbanczyk , the problem was that I had one Compound only in objects, now Codecov passes.
What I don't understand though, is that initially I simply copied the original test, which has also one Compound only when I print `objects1.vals()` before combine is used.

**Comment by adam-urbanczyk** [comment]
You touched this line and that's why it was taken into account in the delta coverage calculation.

**Comment by adam-urbanczyk** [comment]
BTW: is this ready?

**Comment by bragostin** [comment]
@adam-urbanczyk   yes it is

**Review by adam-urbanczyk — APPROVED** [review]

File: cadquery/cq.py [hunk 1]
```diff
@@ -2855,20 +2853,25 @@ def _cutFromBase(self, obj: Shape) -> "Workplane":
...
         :raises: ValueError if there are no items on the stack, or if they cannot be combined
         :return: a CQ object with the resulting object selected
         """
+
         items: List[Shape] = [o for o in self.objects if isinstance(o, Shape)]
         s = items.pop(0)
 
         if items:
-            s = s.fuse(*items)
+            s = s.fuse(*items, glue=glue, tol=tol)
```
Comment (jmwright) [inline cadquery/cq.py:2875]: Having the tol parameter here won't cause problems when glue is false will it?

**Review by jmwright — APPROVED** [review]
Thanks @bragostin !

File: cadquery/cq.py (hunk 1 above)
Comment (adam-urbanczyk) [reply cadquery/cq.py:2875]: They are orthogonal, so no issue.

**Comment by adam-urbanczyk** [comment]
Merging, thanks for the contribution @bragostin !

**Pull request merged**